src/builder-context.h

```diff
@@ -29,6 +29,11 @@
 
 G_BEGIN_DECLS
 
+typedef enum {
+  BUILDER_AS_URL_POLICY_PARTIAL = 0,
+  BUILDER_AS_URL_POLICY_FULL
```

Comment:
```suggestion
  BUILDER_AS_URL_POLICY_FULL,
```
Avoids having to change the line when a new value is added